src/app/objectdetails/objectdetails.component.ts

```diff
@@ -44,9 +44,9 @@ export class ObjectdetailsComponent implements OnInit {
   _showSubmitButton = false
   _loading = false
 
-  private _lsc = false;
   /** Currently selected objectID */
   private objectIdObservable: Observable<string>;
+  orderTypeStr: string;
```

Comment:
I'm reasonably confident that there is a way to properly use the enum. I request to change this to the enum usage, otherwise one spends some time to investigate why this value is kept twice in different types like I did.